invisible_cities/evm/event_model.py

```diff
@@ -375,6 +377,7 @@ def store(self, table):
             row["Qc"      ] = hit .Qc
             row["Ec"      ] = hit .Ec
             row["track_id"] = hit .track_id
+            row["Ep"      ] = hit .Ep
```

Comment:
maybe change positions of track_id and Ep?

Reply:
ok